tendrl/gluster_integration/manager/__init__.py

```diff
@@ -78,6 +80,39 @@ def shutdown(signum, frame):
             {"message": "Signal handler: stopping"}
         )
         complete.set()
```

Comment:
complete.set() should be at the end of this function

Reply:
done